modules/microshift-greenboot-testing-workload-script.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-test-workload-health-check-script_{context}"]
+= Testing a workload health check script
+
+.Prerequisites
+
+* You have root access.
+* You have installed a workload.
+* You have created a health check script for the workload.
+* The {product-title} service is enabled.
+
+.Procedure
+
+. To test a health check script file, first stop {product-title} services and reboot the system by running the following commands:
++
+[source, terminal]
+----
+
+----
+// Q: Do we need to stop MicroShift first? And/or reboot?
```

Comment:
It is not necessary to stop MicroShift before restarting the host.

It is possible to test the scripts without rebooting by just running the script directly and looking at the output.

The best way to verify that _greenboot_ is running the scripts and getting the right output is to reboot the host.